dotnet/src/Connectors/Connectors.Memory.Weaviate/Http/ApiSchema/SearchVectorsResponse.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Nodes;
+
+namespace Microsoft.SemanticKernel.Connectors.Memory.Weaviate.Http.ApiSchema;
```

Comment:
could you split the file pls, one class per file, matching filename and class name?